### DISCUSSION TITLE
Conbee 3 is very far from be usable

### DISCUSSION THREAD

**Discussion by emandtf** [Koenkk/zigbee2mqtt#20554] [Devices]
After 3 weeks of running Z2M (using Dev branch) with Conbee 3 (latest FW of 8th December) stick, I choosed to return back to Conbee 2 because many of my sensors won't work or it's not possible to change their settings.

This is the summary:
- few sensors doesn't accept "pair with Coordinator" directly but works if routed via another device
- few sensors won't complete the Pairing procedure and remains with a red flag. Do a re-configure seems does nothing.
- battery sensors (especially my Aqara P1 motion sensor or Tuya-based temp sensors) that usually need to be waken up by pressing its button immediately before sending the command, won't receive any Setting from Z2M. The error is "_no response (%counter%)_". Switching to Conbee 2 and trying do same thing works flawless.
- few sensors often losts connection to Coordinator/Z2M even if QLI is always 255 (the distance is less than 5 meters with only a wooden door between them) but return "Online" few hours later. Those are "power plugs" with Reporting time every few seconds, so I'm expecting a new data very often.

I hope I'll can use Conbee 3 in the near future, but at this moment is quite unusable :(

**Comment by mendes588** [comment]
Can confirm the issue with some devices (P1, the aqara TRV, etc.) . Some other devices work fine and actually faster than Conbee II.
